target/linux/ramips/image/mt76x8.mk

```diff
@@ -63,6 +84,17 @@ define Device/buffalo_wcr-1166ds
 endef
 TARGET_DEVICES += buffalo_wcr-1166ds
 
+define Device/creality_wb-01
+  IMAGE_SIZE := 16064k
+  IMAGES += cxsw_update.tar.bz2
```

Comment:
I'd still call this factory.bin, or at least factory.tar.bz2.

The name of the images should indicate the use from an OpenWrt perspective, i.e. factory means it's for initial flashing.
Since renaming the image will be necessary anyway, one can also rename from factory.* to the proper name then.